sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/implementation/EventHubReactorConnectionTest.java

```diff
@@ -158,6 +154,24 @@ public void getsManagementChannel() {
         StepVerifier.create(connection.getManagementNode())
             .assertNext(node -> Assertions.assertTrue(node instanceof ManagementChannel))
             .verifyComplete();
+
+    }
+
+    @Test
+    public void createSendLinkTest() {
+        final EventHubReactorAmqpConnection connection = new EventHubReactorAmqpConnection(CONNECTION_ID,
+            connectionOptions, "event-hub-name", reactorProvider, handlerProvider, tokenManagerProvider,
+            messageSerializer, product, clientVersion);
+        Assertions.assertNotNull(connection.createSendLink("send-name", "test-entity-path", new AmqpRetryOptions()));
```

Comment:
`createSendLink` is an async operation and will always return a non-null `Mono` object which should be subscribed to verify that the link was created. See the unit test for `getsManagementChannel`. Something similar to this has to be done:
```java
 StepVerifier.create(connection.createSendLink())
            .assertNext(link -> Assertions.assertTrue(link instanceof ReactorSender))
            .verifyComplete();
```